packages/client/src/devices/InputMediaDeviceManager.ts

```diff
@@ -78,6 +78,7 @@ export abstract class InputMediaDeviceManager<
       return;
     }
 
+    this.state.prevStatus = this.state.status;
```

Comment:
this change is not really necessary, but spotted that prevState was never set when doing enable()

we dont have a usecase for prevState being disable and currentstate being enable yet